CHANGELOG.md

```diff
@@ -20,6 +20,9 @@ Additional documentation and release notes are available at [Multiplayer Documen
 ### Cleanup
 * Removed ParrelSync from the project
 
+### Fixed
+* Fix a Healer ability doesn't work (#893) 
```

Comment:
After linking the PR number, we've tried to give a bit more context on the significance of the change and/or what has been the approach. Could we give a bit more context here on what may have been not working and potentially what was the high-level approach to the fix?

Reply:
Yes, of course.
I've been working on it.
If you have any requests, let me know.